content/src/content/jcr_root/apps/core/wcm/components/commons/editor/clientlibs/panelselect/js/panelselect.js

```diff
@@ -444,6 +444,10 @@
                 subTitle = item[PN_PANEL_TITLE];
             } else if (item.title) {
                 subTitle = item.title;
+            } else if (item.label && item.label.value) {
```

Comment:
No, it's not specific to custom component. This is part of the headless specification of the forms core components. For instance  look at [1].
[1]: https://aemcomponents.dev/content/core-components-examples/library/adaptive-form/emailinput.html#tabs-13ca5be0c7-item-46a8eee3d4-tab

Reply:
That's what I mean, it's specific to Forms Components, unrelated to Core WCM Components.
You can try fixing this in this in the exporter of com.adobe.cq.forms.core.components.internal.models.v1.form.PanelImpl in a similar way to Core WCM Components, see [1], [2], [3].

[1] https://github.com/adobe/aem-core-wcm-components/blob/main/bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/AbstractPanelContainerImpl.java#L119
[2] https://github.com/adobe/aem-core-wcm-components/blob/main/bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/PanelContainerItemImpl.java#L152
[3] https://github.com/adobe/aem-core-wcm-components/blob/main/bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/PanelContainerItemImpl.java#L196

Reply:
This will add an extra property  `cq:panelTitle` in model json which will not adhere headless specification of forms. Other way would be to override this implementaion in Forms (it will require exposing class to namespace i.e `CQ.CoreComponents.PanelSelector` ) 
Does `cq:` prefix for wcm component mean some internal fields ?
cc: @rismehta

Reply:
@LSantha , in the context of forms, we follow a specific specification, and it's important that the JSON aligns with this specification. While we already align with certain WCM properties like :items and :itemsOrder, which are part of the public core component documentation, it's worth noting that cq:panelTitle is an internal WCM property. Therefore, I believe there might be a need for a client-side hook to override this panel title behavior since it's not part of any specification.

Reply:
@LSantha I have removed forms-specific change. Basically I have exposed `PanelSelector` to `CQ.CoreComponents` namespace, it is already mentioned in [docs](https://github.com/adobe/aem-core-wcm-components/blob/3524e33424bf6831faca014284c5bad803d3744d/content/src/content/jcr_root/apps/core/wcm/components/commons/editor/clientlibs/panelselect/js/panelselect.js#L35) but was exposed.
Please review and merge.

Reply:
Thanks @dmaurya929 , LGTM.